src/main/resources/default/assets/tycho/styles/sirius-colors.scss

```diff
@@ -42,8 +42,21 @@ $sirius-cyan-light: #68D3E3;
 $sirius-deep-blue: #53599A;
 $sirius-deep-blue-dark: #352E87;
 
+$sirius-white: #FFFFFF;
+$sirius-white-dark: #F4F4F4;
+
+$sirius-gray-lighter: #E6E6E6;
+$sirius-grey-lighter: $sirius-gray-lighter;
+$sirius-gray-light: #C7C7C7;
+$sirius-grey-light: $sirius-gray-light;
 $sirius-gray: #ACACAC;
+$sirius-grey: $sirius-gray;
 $sirius-gray-dark: #808080;
+$sirius-grey-dark: $sirius-gray-dark;
+$sirius-gray-darker: #474747;
+$sirius-grey-darker: $sirius-gray-darker;
+
+$sirius-black: #000000;
```

Comment:
Vllt nochmal fragen ob schwarz wirklich schwarz sein sollte oder eher die dark Töne ausm Style Guide dafur gedacht sind. Oft nimmt man ja nicht 100% Schwarz.

Reply:
we define a text-muted in core, don't we? maybe add this color here aswell

Reply:
We do. 😊 The color is `#6c757d`, a slightly tinted grey. However, I am running out of names for the greys. We could also use a different grey for the muted text, like `$sirius-gray-dark` (`#808080`). In general, we have more greys in the brighter range than in the darker one. Another shade that we don't have here is `#212529` for "soft" text.

Reply:
Die Farbfrage übergebe ich nochmal @Matina2402, merge diesen PR aber im Sinne der Fertigstellung des Sprints